vm/src/vm.rs

```diff
@@ -823,6 +841,44 @@ impl VirtualMachine {
                 self.push_block(Block::TryExcept { handler: *handler });
                 None
             }
+            bytecode::Instruction::SetupWith { end } => {
+                let obj = self.pop_value();
+                // Call enter:
+                match self.call_method(obj, "__enter__", vec![]) {
+                    Ok(manager) => {
```

Comment:
Not a big deal, but I think the naming is a bit off here; `obj` is the context manager here, not what `__enter__` returns. I'm not sure I can think of a good name for what `__enter__` returns. (Perhaps just `context`?)